kotlin/delete-features-feature-service/README.md

```diff
@@ -1,31 +1,31 @@
-# Delete features (Feature Service)
+# Delete features (feature service)
 
 Delete features from an online feature service.
 
-![Delete Features (Feature Service)](delete-features-feature-service.png)
+![Image of delete features feature service](delete-features-feature-service.png)
 
 ## Use case
 
-A `FeatureLayer` created using this `ServiceFeatureTable` that is applied to the `ArcGISMap` will display existing Features and allow a user to delete individual features by tapping on them.
+Sometimes users may want to delete features from an online feature service.
+
+## How to use the sample
+
+Tap on a feature on the map and tap the trash icon to delete it.
 
 ## How it works
 
-1. Create a new `ServiceFeatureTable` from a URL.
-1. Create a `FeatureLayer` from the `ServiceFeatureTable` created in step 1.
-1. Detect a tap on a `Feature` and display a `Callout` by invoking `MapView.getCallout()`.
-1. Detect a tap on the contents of the `Callout`, asking the user to confirm they wish to delete the `Feature`.
-1. Get the `Feature` by querying the `FeatureTable` to obtain the `Feature` instance, invoking `FeatureLayer.getFeatureTable().queryFeaturesAsync()`.
-1. Delete the `Feature` from the `ServiceFeatureTable`, `ServiceFeatureTable.deleteFeatureAsync(Feature)`.
-1. Update the new feature to the server, `ServiceFeatureTable.applyEditsAsync()`.
+1. Create a `ServiceFeatureTable` object from a URL.
+2. Create a `FeatureLayer` object from the `ServiceFeatureTable`.
+3. Select features from the `FeatureTable` via `queryFeaturesAsync()`.
```

Comment:
should probably remove the brackets after these method calls since we're using Kotlin

Reply:
Is that a thing? I didn't realize!